plugins/lookup/passwordstore.py

```diff
@@ -25,9 +25,9 @@
         env:
           - name: PASSWORD_STORE_DIR
       create:
-        description: Create the password if it does not already exist.
+        description: Create the password if it does not already exist. Takes precedence over C(missing).
         type: bool
-        default: 'no'
+        default: True
```

Comment:
```suggestion
        default: false
```
`no` is `false`, not `true`.

Reply:
Ouch.. coffee first, refactoring after; not the other way around